src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -168,15 +183,24 @@ protected override bool Receive(object message)
                     _commitRefreshing.Committed(committed.Offsets);
                     return true;
                 
-                case KafkaConsumerActorMetadata.Internal.Stop stop:
+                case KafkaConsumerActorMetadata.Internal.Stop _:
                     _log.Debug($"Received Stop from {Sender}, stopping");
                     Context.Stop(Self);
                     return true;
                 
+                case KafkaConnectionFailed kcf:
+                    ProcessError(kcf);
+                    Self.Tell(KafkaConsumerActorMetadata.Internal.Stop.Instance);
```

Comment:
So if there is a temporary disconnect from broker, there is no other way of handling that except stopping consuming? Right now this depends on supervision strategy, you can just try to resume. ProcessError method already sending failure notification to parent (source stage), and parent may stop this actor if `Dicider` is configured to do that.